extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -0,0 +1,123 @@
+  position: fixed !important;
+  z-index: 2147483645 !important;
+  width: 100%;
+  box-sizing: border-box;
+  margin: 0;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+/**
+ * The following rule is needed to reset the stacking context.
+ * By resetting the properties that create new stacking context on the
+ * ancestors of the item being lightboxed, we end up with a single stacking
+ * context and therefore the z-index will be absolute. See https://goo.gl/uqY5CN
+ * Code is copied from Webkit's FullScreen Css https://goo.gl/gjbKoI
+ */
+.-amp-lightboxed-ancestor {
+  z-index: auto !important;
+  position: static !important;
+  opacity: 1 !important;
+  transform: none !important;
+  clip: none !important;
+  transition: none !important;
+  will-change: auto !important;
+  -webkit-mask: none !important;
+  -webkit-filter: none !important;
+  -webkit-box-reflect: none !important;
+  -webkit-perspective: none !important;
+  -webkit-transform-style: flat !important;
+  /* TODO(aghassemi): Add MS and Moz support */
+}
+:root.-amp-lightboxed-ancestor {
+  overflow: hidden !important;
+}
+
+/* Page Mask */
+.-amp-lightbox-viewer-mask {
+  position: fixed !important;
+  z-index: 2147483644 !important;
+  top:0 !important;
+  left: 0 !important;
+  right: 0 !important;
+  bottom: 0 !important;
+  background-image: none !important;
+  opacity: 1 !important;
+  background-color: rgba(0,0,0, 1) !important;
+}
+
+/* Controls */
+.amp-lightbox-viewer-button-next,
+.amp-lightbox-viewer-button-previous,
+.amp-lightbox-viewer-button-close {
+  z-index: 2147483646 !important;
+  position: fixed !important;
+  width: 50px;
+  height: 50px;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-color: rgba(0,0,0,0.7);
+}
+/* TODO(aghassemi): Polish the controls and add RTL support */
+.amp-lightbox-viewer-button-next,
+.amp-lightbox-viewer-button-previous {
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.amp-lightbox-viewer-button-next {
+  right: 0;
+  background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="%23fff" viewBox="0 0 18 18"><path d="M9 3L7.94 4.06l4.19 4.19H3v1.5h9.13l-4.19 4.19L9 15l6-6z" /></svg>');
+}
+
+.amp-lightbox-viewer-button-previous {
+  left: 0;
+  background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="%23fff" viewBox="0 0 18 18"><path d="M15 8.25H5.87l4.19-4.19L9 3 3 9l6 6 1.06-1.06-4.19-4.19H15v-1.5z" /></svg>');
+}
+
+.amp-lightbox-viewer-button-close {
+  top: 0;
+  left: 0;
+  background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" fill="%23fff" height="24" viewBox="0 0 24 24" width="24"><path d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/><path d="M0 0h24v24H0z" fill="none"/></svg>');
+}
+
+.-amp-lightbox-viewer[no-next] .amp-lightbox-viewer-button-next {
```

Comment:
We currently as a rule just use `amp-disabled` class. By default we hide elements with `opacity: 0; visibility: hidden;`, but this way we allow easier customization via CSS.